test_mpop_service/TestMpopService.cpp

```diff
@@ -521,32 +521,61 @@ void TestMpopService::test_24_getAnswerByGender() {
 
     // User Questions
     static const QString TEST_QUESTION_07_ID = "equitable_riches";
+    static const QString TEST_QUESTION_08_ID = "equitable_minorites_culturelles";
+    static const QString TEST_QUESTION_09_ID = "confiance_systeme";
 
     // User Answers
     static const int TEST_QUESTION_07_VALUE = 55;
+    static const int TEST_QUESTION_08_VALUE = 65;
+    static const int TEST_QUESTION_09_VALUE = 67;
+
 
     // Calculated Avg. for three User
     static const int Test_QUESTION_07_AVG = 55;
+    static const int Test_QUESTION_08_AVG = 65;
+    static const int Test_QUESTION_09_AVG = 67;
 
     // declaration of RFIDs
     static const QString TEST_RFID_TAG_1 = "test_RFID_tag_00011";
+    static const QString TEST_RFID_TAG_2 = "test_RFID_tag_00012";
+    static const QString TEST_RFID_TAG_3 = "test_RFID_tag_00013";
```

Comment:
We should make sure to delete all those users and their answers, once we are done with the unit tests.